### ISSUE TITLE
Alternatives to GOTO?

### ISSUE THREAD

**Issue by springmeyer** [mapbox/ci-scripts#1]
@BergWerkGIS I find the use of GOTO in the .bat scripts hard to follow (and generally a programming construct I see rarely). Can you take a pass at refactoring the scripts to use IF/Else statements? Or do you feel like the limitations of IF/Else in bash warrant the use of GOTO? I'm okay keeping it if you feel strongly, but would prefer an alternative solution.

**Comment by wilhelmberg** [comment]
I strongly lean towards keeping `GOTO`.
`IF/ELSE` is a real PITA within batch files.

Please reopen if you think otherwise.

**Comment by springmeyer** [comment]
🆗
